frontend/src/app/leaderboard/top3.tsx

```
@@ -0,0 +1,25 @@
+import React from "react";
+
+const top3 = () => {
+  return (
+    <div className="flex w-full mt-10 gap-5 justify-center">
+      <div className="avatar">
+        <div className="w-16 md:w-24 mt-5 rounded-full">
+          <img src="https://img.daisyui.com/images/profile/demo/2@94.webp" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)